### ISSUE TITLE
[BUG] Input working in UI but not work in game

### ISSUE THREAD

**Issue by vedmysh** [longjunyu2/winlator#104]
**Device Information**
- Device Model: ASUS ROG 8 Pro
- Device Processor Model: Qualcomm Snapdragon 8 gen 3
- Android Version: 14
- Kernel Version: 6.1.68-android14-11-g9a550d45f6db-ab12075393

**Application Configuration Information**
- App Version: 7.1.3
- Display Driver: Turnip-24.3.0
- dxwrapper: DXVK-2.3.1
- box64 Version: 0.2.8
- box64 Preset: Compatibility

**Description of the Issue**
After starting container Input Controls works - for example selecting RTS profile and clicking any keys produce apropriate output. But after game start (tested on Witcher 3 and Dying Light) it looks like no "keypress" propogated to the game.

**Comment by longjunyu2** [comment]
@vedmysh

I tested The Witcher 3, and I don't have this issue. Please check if you used VirtualGamepad before RTS, or if your device is connected to a game controller.

If it's the former, please avoid using VirtualGamepad, as the game will think you're using a game controller. If it's the latter, please disconnect the controller or ensure that `Enable XInput` and `Enable DInput` are not checked.

**Comment by vedmysh** [comment]
Started container. Selected Virtual Gamepad. It works - at least it works in the contol panel\input devices\xinput tab - i see axis movement and keypress. Now starting the game. It shows some text and ask to press X. I press X many times, but nothing happends. So the symptoms are same no matter what virtual onscreen mapping I use...

Do you test it on the same ASUS ROG Phone 8 Pro? No special settings?

**Comment by longjunyu2** [comment]
@vedmysh try setting `Startup Selection` to `Aggressive`

**Comment by vedmysh** [comment]
> @vedmysh try setting `Startup Selection` to `Aggressive`

It works! Thank you! Tested in Dying Light.

**Comment by roninspin** [comment]
I have the same problem (both virtual and physical gamepad). Wine's "Input devices" indicates axes and buttons when I press them, but all my games don't react at all. The strange thing is that a couple of months ago I installed winlator 7.1.2 glibc (IIRC) and it worked then. Now I've tried different Winlator versions (with clean installation, too), different settings, Nothing works.
P.S. My controller works in games when I use wine in chroot,
